fuel-android/src/test/kotlin/com/github/kittinunf/fuel/android/BaseTestCase.kt

```diff
@@ -1,21 +1,38 @@
 package com.github.kittinunf.fuel.android
 
+import org.junit.After
+import org.junit.Before
 import org.junit.runner.RunWith
 import org.robolectric.RobolectricTestRunner
 import org.robolectric.annotation.Config
 import java.util.concurrent.CountDownLatch
 import java.util.concurrent.TimeUnit
 
 @RunWith(RobolectricTestRunner::class)
-@Config(constants = BuildConfig::class, sdk = intArrayOf(26))
+@Config(constants = BuildConfig::class, sdk = [26])
 abstract class BaseTestCase {
 
-    val DEFAULT_TIMEOUT = 15L
+    companion object {
+        private const val DEFAULT_TIMEOUT = 15L
+    }
 
+    protected lateinit var mock: MockHelper
     lateinit var lock: CountDownLatch
 
     fun await(seconds: Long = DEFAULT_TIMEOUT) {
         lock.await(seconds, TimeUnit.SECONDS)
     }
 
+
+    @Before
+    fun setup() {
+        this.mock = MockHelper()
```

Comment:
do you need `this`?

Reply:
Does it work without? Sorry I think I might be mixing languages here :). @markGilchrist what are the rules in Kotlin when you must prefix with this? Is it like ruby (only when you have a conflicting identifier, or a "private" setter?)

Reply:
https://kotlinlang.org/docs/reference/this-expressions.html It can work without it.

Reply:
@SleeplessByte  sorry for my late reply, you only need this if the compiler can't work out which variable you are taking about

```kotlin
fun setUser(user:User){
    this.user = user
    user = user // the compiler thinks you are assigning the variable to itself
}
```

Reply:
Thanks! Same as in Ruby then, I can work with this :) (well without `this` :P)